inf-ruby.el

```diff
@@ -301,8 +321,11 @@ run)."
 
 ;;;###autoload
 (defun run-ruby (&optional command name)
-  "Run an inferior Ruby process, input and output via buffer `*NAME*'.
-If there is a process already running in `*NAME*', switch to that buffer.
+  "Run an inferior Ruby process, input and output via buffer
```

Comment:
Could you make the docstring's first sentence to still take one line? It's the Emacs style guide requirement.

Also, "dedicated to ... or file" is kinda awkward: I don't we'll have any buffers dedicated to a particular file.